db/schema.rb

```diff
@@ -69,15 +69,6 @@
     t.index ["user_id"], name: "index_listings_on_user_id"
   end
 
```

Comment:
@zuccamia @worksofgenius I am not sure what I did here, did I just delete the file...? What is going on...?